internal/services/nginx/nginx_deployment_resource.go

```diff
@@ -260,6 +253,17 @@ func (m DeploymentResource) Arguments() map[string]*pluginsdk.Schema {
 		"tags": commonschema.Tags(),
 	}
 
+	if !features.FivePointOhBeta() {
+		resource["managed_resource_group"] = &pluginsdk.Schema{
+			Deprecated:   "The `managed_resource_group` field has been deprecated and will be removed in v5.0 of the AzureRM Provider.",
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			Computed:     true,
+			ForceNew:     true,
```

Comment:
We should remove `ForceNew` on this so that users can remove this from their configs without triggering a recreation of the resource 
```suggestion
```